src/Symfony/Bundle/FrameworkBundle/Routing/Loader/Configurator/RouteConfigurator.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Routing\Loader\Configurator;
+
+use Symfony\Bundle\FrameworkBundle\Controller\RedirectController;
+use Symfony\Bundle\FrameworkBundle\Controller\TemplateController;
+use Symfony\Bundle\FrameworkBundle\Routing\Loader\Configurator\Traits\AddTrait;
+use Symfony\Component\Routing\Loader\Configurator\RouteConfigurator as BaseRouteConfigurator;
+
+/**
+ * @author Jules Pietri <jules@heahprod.com>
+ */
+class RouteConfigurator extends BaseRouteConfigurator
+{
+    use AddTrait;
+
+    /**
+     * @param string $template The template name
+     */
+    final public function template(string $template): TemplateRouteConfigurator
```

Comment:
will need to account for https://github.com/symfony/symfony/pull/35257 :)

Reply:
#35257 merged, can be rebased now.

Reply:
Done.